src/style/internal/z-index-variables.scss

```diff
@@ -9,3 +9,4 @@ $limel-slider--thumb-container--z-index: 1 !default;
 $limel-tab-bar--active-tab--z-index: 2 !default;
 $limel-table--has-interactive-rows--selectable-row--hover--z-index: 1 !default;
 $limel-popover-before--z-index: -1 !default;
+$limel-button-group-radio-button-keyboard-focused: 1 !default;
```

Comment:
When I created all these z-index variables, I added `--z-index` at the end of their names. I was thinking that it's nice to always know from the name of a variable that it's a z-index. But maybe that's unnecessary, since they'll probably be seen by the reader either in this file, or at the end of a line beginning with `z-index:`… 🤔

I guess if they're sent in to a mixin, it could be useful.

Anyway, I just wanted to check if leaving off the `--z-index` suffix was intentional or not? 😊

Reply:
Oh no I simply missed it. I have nothing agains having `--z-index` I'll do a fixup in a min